website/client/components/groups/group.vue

```diff
@@ -460,20 +461,40 @@ export default {
     acceptCommunityGuidelines () {
       this.$store.dispatch('user:set', {'flags.communityGuidelinesAccepted': true});
     },
-    load () {
+    async load () {
       if (this.isParty) {
         this.searchId = 'party';
         // @TODO: Set up from old client. Decide what we need and what we don't
         // Check Desktop notifs
         // Load invites
       }
-      this.fetchGuild();
-
+      await this.fetchGuild();
+      // Fetch group members on load
+      this.members = await this.loadMembers({
+        groupId: this.group._id,
+        includeAllPublicFields: true,
+      });
       this.$root.$on('updatedGroup', group => {
         let updatedGroup = extend(this.group, group);
         this.$set(this.group, updatedGroup);
       });
     },
+
+    /**
+     * Method for loading members of a group, with optional parameters for
+     * modifying requests.
+     *
+     * @param {Object}  payload     Used for modifying requests for members
+     */
+    async loadMembers (payload = null) {
```

Comment:
If the method contains await it is a promise and you do need async.

Reply:
yes. But only if you need to do something after the promise is done, here it doesn't has to do anything else, it would just return a new promise, and the caller still can await this if needed

Reply:
Ah, I misread. You said remove both. Nvm

Reply:
Sorry for the delay with the fix - but I've removed both the unnecessary `async` and `await` usage in the `loadMembers` methods.

Let me know if there's anything else!

Reply:
actually, looking above you are awaiting load members. So although negue's second comment is correct, the first is still not. You still need await even if nothing happens after it in the method. 

If you are waiting for a promise to finish, you need await. In this case the load members dispatch is a promise and you do want to wait for it to finish so the user can see the new members. Sorry, I should of read the whole PR.

Reply:
I actually tested this before committing it - it seemed to be functioning just fine, so I think @negue was correct. I believe that the `await` on `this.loadMembers` inside of `load()` will wait on `loadMembers` to finish, but does not require `loadMembers` itself to utilize `async` / `await`.

I can change it back if you feel otherwise, as it might behave differently in production vs my local dev environment!

Reply:
Eh... Okay nvm. Just forget I committed here. I am just completely misreading this from my phone.

Reply:
😄 No worries! Thanks for chiming in!